backend/server.js

```diff
@@ -84,6 +84,46 @@ app.post(
     },
 );
 
+// FILE server.js
+
+// Route for getting json containing all relevant user data
+app.get('/users/:userId', function(req, res) {
+  console.log('test');
+  usersModel.findOne({userId: req.params.userId}, function(err, userData) {
+    if (err) {
+      res.send(err);
+    } else {
+      const relevantData = {
+        email: userData.email,
+        username: userData.username,
+      };
+      res.send(relevantData);
+    }
+  });
+});
+
+// Route for getting username
+app.get('/users/:userId/username', function(req, res) {
+  usersModel.findOne({userId: req.params.userId}, function(err, userData) {
+    if (err) {
+      res.send(err);
+    } else {
+      res.send(userData.username);
+    }
+  });
+});
+
+// Route for getting user email
```

Comment:
Really nice you made multiple routes to get different data, makes the code easily changeable if we add new fields to the db.